src/lib/mongoBackend/mongoUpdateSubscription.cpp

```diff
@@ -0,0 +1,608 @@
+/*
+*
+* Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
+*
+* This file is part of Orion Context Broker.
+*
+* Orion Context Broker is free software: you can redistribute it and/or
+* modify it under the terms of the GNU Affero General Public License as
+* published by the Free Software Foundation, either version 3 of the
+* License, or (at your option) any later version.
+*
+* Orion Context Broker is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+* General Public License for more details.
+*
+* You should have received a copy of the GNU Affero General Public License
+* along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+*
+* For those usages not covered by this license please contact with
+* iot_support at tid dot es
+*
+* Author: Fermin Galan
+*/
+
+#include <map>
+#include <string>
+#include <vector>
+
+#include "logMsg/logMsg.h"
+#include "logMsg/traceLevels.h"
+#include "rest/OrionError.h"
+#include "alarmMgr/alarmMgr.h"
+#include "apiTypesV2/SubscriptionUpdate.h"
+#include "mongoBackend/connectionOperations.h"
+#include "mongoBackend/MongoGlobal.h"
+#include "mongoBackend/MongoCommonSubscription.h"
+#include "mongoBackend/dbConstants.h"
+#include "mongoBackend/safeMongo.h"
+#include "mongoBackend/mongoSubCache.h"
+#include "common/defaultValues.h"
+
+#include "cache/subCache.h"
+
+#include "mongo/client/dbclient.h"
```

Comment:
Ok. Is that mentioned in the style guide that 3rd party goes after STL-and-the-like includes?